src/components/dynamic/admin/services/calendar/Events.jsx

```diff
@@ -17,6 +17,19 @@ const CalendarEvents = () => {
   const [date, setDate] = useState(new Date());
   const [view, setView] = useState("month");
 
+  const handleShortcuts = (e) => {
+    switch (e.keyCode) {
```

Comment:
there is there a way to match it based on character like if e.character === "w" so we dont need to know the ASCII codes

Reply:
so would i use `e.code === "KeyW"` or `e.key === "w" || e.key === "W"`

Reply:
`e.code === "KeyW"` or just `e.key === "w"`?